composer.json

```diff
@@ -24,7 +24,9 @@
     },
     "require-dev": {
         "league/oauth2-facebook": "^1.1",
-        "symfony/security-guard": "^2.8|^3.0"
+        "symfony/security-guard": "^2.8|^3.0",
+        "phpunit/phpunit": "^4.8",
```

Comment:
I recommend sticking to the single major for the universally reproducible results. As dev requirements are installed per project this means it's fine if you have 3 projects on the same machine depending on PHPunit `^4.6`, `5.1.*` and `^5.2` - each will get their own copy.

As dev requirements themselves are never propagated to external projects you can in practice do whatever the hell you want with them, it's your project  😄   `php-code-coverage` itself has been installed [25.5 million times](https://packagist.org/packages/phpunit/php-code-coverage) so I'd think it enough precedent   😉 

I'll pick this up along with #36, do expect a freakish PR coming up from that with all files modified hehe.